src/components/Auth/Login/Login.react.js

```diff
@@ -192,9 +194,13 @@ class Login extends Component {
       }
       case 'password': {
         const password = event.target.value.trim();
+        const passwordLength = password.length < 6;
```

Comment:
Again, the same issue. Does passwordLength mean true/false if the length of password is less than 6 or just the length of the password ? Naming of variable shouldn't be ambiguous.